docker-compose.yml

```diff
@@ -1,6 +1,7 @@
 services:
   frontend:
     platform: linux/amd64
+    image: umasscds/inatator_frontend:0.1.0
```

Comment:
If you're going to start adding version numbers here, we should start a Changelog file as well, since this repo doesn't have one yet. I like the templates from https://keepachangelog.com/en/1.0.0/

Reply:
Added in [separate](https://github.com/UMassCDS/ds4cg2024-inaturalist/pull/35) PR